addons/gforces/functions/fnc_pfhUpdateGForces.sqf

```diff
@@ -13,26 +12,18 @@
  */
  #include "script_component.hpp"
 
-private ["_interval", "_player", "_newVel", "_accel", "_currentGForce", "_average", "_sum", "_classCoef", "_suitCoef", "_gBlackOut", "_gRedOut", "_g", "_gBO", "_coef", "_strength"];
- 
-EXPLODE_2_PVT(_this,_params,_pfhId);
-
-_interval = ACE_time - GVAR(lastUpdateTime);
 
-// Update the g-forces at constant game ACE_time intervals
-if (_interval < INTERVAL) exitWith {};
+ if (isNull ACE_player) exitWith {};
 
-if (isNull ACE_player) exitWith {};
+ if !(alive ACE_player) exitWith {};
 
-if !(alive ACE_player) exitWith {};
+private ["_player", "_newVel", "_accel", "_currentGForce", "_average", "_sum", "_classCoef", "_suitCoef", "_gBlackOut", "_gRedOut", "_g", "_gBO", "_coef", "_strength"];
 
 GVAR(lastUpdateTime) = ACE_time;
-
-/*if !(vehicle ACE_player isKindOf "Air") exitWith {
```

Comment:
If we are in the middle of a blackout, and eject from a air vehicle, the effect will never go away.